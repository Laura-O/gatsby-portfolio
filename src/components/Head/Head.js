import React from 'react';
import Helmet from 'react-helmet';

export default function Head() {
  return (
    <Helmet
      title="Laura Ohrndorf - Portfolio"
      meta={[
        { name: 'description', content: 'Sample' },
        { name: 'keywords', content: 'portfolio, something' },
      ]}
    >
      <meta name="twitter:site" content="@laura-fyi" />
      <meta name="og:type" content="website" />
      <meta name="og:site_name" content="Laura Ohrndorf" />
      <html lang="en" />
    </Helmet>
  );
}
