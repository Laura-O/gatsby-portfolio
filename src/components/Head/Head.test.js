import React from 'react';
import { shallow } from 'enzyme';
import Head from './Head';

test('rendering', () => {
  const wrapper = shallow(<Head path="/something" />);
  expect(wrapper).toMatchSnapshot();
});
