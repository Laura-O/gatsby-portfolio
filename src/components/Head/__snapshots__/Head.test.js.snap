// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<HelmetWrapper
  defer={true}
  encodeSpecialCharacters={true}
  meta={
    Array [
      Object {
        "content": "Sample",
        "name": "description",
      },
      Object {
        "content": "portfolio, something",
        "name": "keywords",
      },
    ]
  }
  title="Laura Ohrndorf - Portfolio"
>
  <meta
    content="@laura-fyi"
    name="twitter:site"
  />
  <meta
    content="website"
    name="og:type"
  />
  <meta
    content="Laura Ohrndorf"
    name="og:site_name"
  />
  <html
    lang="en"
  />
</HelmetWrapper>
`;
